options.html

```diff
@@ -35,7 +35,21 @@ <h1>Reload All Tabs <em id="version"></em></h1>
       </div>
     </div>
     <div class="extension-template">
-      <div class="extension-header">Context Menu (Right Click)</div>
+      <div class="extension-header">Menu Button (Default)</div>
+      <div class="extension-options">
+        <dl>
+		  <dt>When clicking the menu, reload</dt>
+		  <dd>
+            <select id="buttonDefaultAction">
+              <option value="allWindows">All Windows</option>
+              <option value="window">Current Window</option>
+              <option value="pinned">Current Window, Pinned tabs</option>
+              <option value="unpinned">Current Window, Unpinned tabs</option>
+            </select>
+          </dd>
+        </dl>
+      </div>
+	  <div class="extension-header">Context Menu (Right Click)</div>

```

Comment:
nit: can you please fix the white space? I see a mix of tabs and spaces here.

Reply:
updated in my fork